src/components/dev-hub/success-state.js

```diff
@@ -11,8 +11,8 @@ const SuccessContainer = styled('div')`
     }
 `;
 
-const SuccessState = ({ children }) => (
-    <SuccessContainer>
+const SuccessState = ({ children, ...props }) => (
+    <SuccessContainer {...props}>
```

Comment:
I also passed `props` here to give some extra margin:

<img width="776" alt="Screen Shot 2020-05-26 at 2 50 27 PM" src="https://user-images.githubusercontent.com/9064401/82938746-474b1080-9f60-11ea-84f5-60188dd9440a.png">